app/assets/scripts/components/charts/horizontal-bar.js

```diff
@@ -74,14 +76,15 @@ var HorizontalBarChart = React.createClass({
         right: margin.left
       });
     }
-    const innerWidth = width - margin.left - margin.right;
-    const innerHeight = height - margin.top - margin.bottom;
+    let innerWidth = width - margin.left - margin.right;
 
     // short circut if we have too small an area
-    if (innerWidth <= 0 || innerHeight <= data.length * 2) {
+    if (innerWidth <= 0) {
```

Comment:
For some reason when you initially toggle out to the chart view, the value here goes negative from the margin values used to calculate innerWidth earlier, and it breaks the load. 

I looked into what was causing the issue, but having this condition seems to work & appears to be the solution used for Map Egypt as well.